cmd/thanos/store.go

```diff
@@ -154,6 +181,57 @@ func runStore(
 		s := grpc.NewServer(defaultGRPCServerOpts(logger, reg, tracer)...)
 		storepb.RegisterStoreServer(s, gs)
 
+		g.Add(func() error {
+			return errors.Wrap(s.Serve(l), "serve gRPC")
+		}, func(error) {
+			l.Close()
+		})
+	} else if s3Config.Validate() {
+		var bkt objstore.Bucket
+
+		bkt, err := s3.NewBucket(s3Config, reg)
+		if err != nil {
+			return errors.Wrap(err, "create s3 client")
+		}
+		bkt = objstore.BucketWithMetrics(*s3Config.Bucket, bkt, reg)
+
+		gs, err := store.NewBucketStore(
+			logger,
+			reg,
+			bkt,
+			dataDir,
+			indexCacheSizeBytes,
+			chunkPoolSizeBytes,
+		)
+		if err != nil {
+			return errors.Wrap(err, "create GCS store")
```

Comment:
error wrap description here is not relevant to GCS anymore. Same with var name `gs`